facdb/sql/dycd_afterschoolprograms.sql

```diff
@@ -0,0 +1,41 @@
+DROP TABLE IF EXISTS _dycd_afterschoolprograms;
+
+SELECT
+    uid,
+    source,
+    site_name as facname,
+    parsed_hnum as addressnum,
+    parsed_sname as streetname,
+    location_1 as address,
+    NULL as city,
+    postcode as zipcode,
+    SPLIT_PART(borough__community, ',', 1) as boro,
+    NULL as borocode,
+    bin,
+    bbl,
+    REPLACE(program, 'NDA Immigrats', 'NDA Immigrants')||' '||program_type as factype,
+    (CASE
+		WHEN program = 'Beacon'
+			OR program = 'Beacon Satellite'
+			OR program = 'High-School Aged Youth'
+			OR program = 'Middle School Youth'
```

Comment:
We might need to be more forgiving with these case statements, because there are records where program = ' Middle School Youth' (leading space at beginning) so it's getting classified as "Youth Centers, Literacy Programs, and Job Training Services"